src/renderer/pages/project/components/script_editor/beat_preview.vue

```
@@ -1,67 +1,105 @@
 </template>
 
 <script setup lang="ts">
-import { FileImage, Video, Play } from "lucide-vue-next";
+import { FileImage, Video, Play, Loader2, Sparkles } from "lucide-vue-next";
+import { computed } from "vue";
 import type { MulmoBeat } from "mulmocast/browser";
 
-import { Loader2 } from "lucide-vue-next";
+import { Button } from "@/components/ui/button";
 import { mediaUri } from "@/lib/utils";
+import { isMediaBeat, isLocalSourceMediaBeat } from "@/lib/beat_util";
```

ESLint

[error] 'isMediaBeat' is defined but never used. Allowed unused vars must match /^__/u.

(@typescript-eslint/no-unused-vars)